internal/cli/completion.go

```diff
@@ -27,18 +28,67 @@ func DefaultCompletionOptions() *CompletionOptions {
 func NewCmdCompletion() *cobra.Command {
 	o := DefaultCompletionOptions()
 	cmd := &cobra.Command{
-		Use:          "completion",
-		Short:        "Generate autocompletion script",
+		Use:   "completion [bash|zsh|fish|powershell]",
+		Short: "Generate shell autocompletion script",
+		Long: `Generate the autocompletion script for flightctl for the specified shell.
+
+Supported shells: bash, zsh, fish, powershell.
+
+To load completions for the current session:
+  - bash:        source <(flightctl completion bash)
+  - zsh:         source <(flightctl completion zsh)
+  - fish:        flightctl completion fish | source
+  - powershell:  flightctl completion powershell | Out-String | Invoke-Expression
+
+To load completions persistently:
+  - bash (Linux):      flightctl completion bash | sudo tee /etc/bash_completion.d/flightctl > /dev/null
+  - bash (macOS/Homebrew): flightctl completion bash > $(brew --prefix)/etc/bash_completion.d/flightctl
+  - zsh:               flightctl completion zsh > ${ZDOTDIR:-$HOME}/.zsh/completions/_flightctl && echo 'fpath+=(${ZDOTDIR:-$HOME}/.zsh/completions); autoload -U compinit; compinit' >> ${ZDOTDIR:-$HOME}/.zshrc
+  - fish:              flightctl completion fish > ~/.config/fish/completions/flightctl.fish
+  - powershell:        flightctl completion powershell > "$HOME/.config/flightctl/completion.ps1"; Add-Content -Path $PROFILE -Value ". $HOME/.config/flightctl/completion.ps1"
```

Comment:
IDK Windows or PowerShell, but I would have expected Windows-style paths (with backslashes) and directories like `%UserProfile%\Documents\PowerShell`?